ports/unix/README.md

```diff
@@ -1,17 +1,61 @@
-The Unix version
-----------------
+MicroPython Unix port
+=====================
+
+The "unix" port runs in standard Unix-like environments including Linux, BSD,
+macOS, and Windows Subsystem for Linux.
 
-The "unix" port requires a standard Unix-like environment with gcc and GNU
-make. This includes Linux, BSD, macOS, and Windows Subsystem for Linux. The
-x86 and x64 architectures are supported (i.e. x86 32- and 64-bit), as well as
-ARM and MIPS. Making a full-featured port to another architecture requires
+The x86 and x64 architectures are supported (i.e. x86 32- and 64-bit), as well
+as ARM and MIPS. Extending the unix port to another architecture requires
 writing some assembly code for the exception handling and garbage collection.
 Alternatively, a fallback implementation based on setjmp/longjmp can be used.
 
-To build (see section below for required dependencies):
+Building
+========
+
+Dependencies
+------------
+
+To build the unix port locally then you will need:
+
+* git command line executable, unless you downloaded a source .tar.xz file from
+  https://micropython.org/download/
+* gcc (or clang for macOS) toolchain
+* GNU Make
+* Python 3.x
+
+To build the default "standard" variant and configuration, then you will also
+need:
+
+* `pkg-config` tool
+* `libffi` library and headers
+
+On Debian/Ubuntu/Mint and related Linux distros, you can install all these
+dependencies with a command like:
+
+```
+# apt install build-essential git python3 pkg-config libbfi-dev
+```
+
+(See below for steps to build either a standalone or minimal MicroPython
+executable that doesn't require system `libffi` or `pkg-config`.)
+
+Default build steps
+-------------------
+
+To set up the environment for building (not needed every time), starting from
+the top-level MicroPython directory:
 
     $ cd ports/unix
+    $ make -C ../../mpy-cross
```

Comment:
A bit less typing to do `make -C mpy-cross` before `cd ports/unix`.

Reply:
I think the trade-off is extra simplicity in having all of the sample commands run from the same (port) directory.